src/module.ts

```diff
@@ -66,24 +69,21 @@ export const plugin = new PanelPlugin<PanelOptions>(PlotlyPanel)
         defaultValue: '',
         category: ['X Axis'],
       })
-      .addCustomEditor({
+      .addMultiSelect<string, SelectFieldConfigSettings<string>>({
```

Comment:
We initially had our own version of a MultiSelect editor, but it's now a part of the standard Grafana library.